### PR TITLE
Set Max Series

### PR DESCRIPTION
**Pull request by rohbharadwaj**

### CODE REVIEW

**Review by timhallinflux — APPROVED** [review]

File: charts/influxdb-enterprise/values.yaml [hunk 1]
```diff
@@ -227,6 +227,7 @@ data:
     # annotations:
     # accessMode: ReadWriteOnce
     # size: 8Gi
+  maxseries: 1500000
```
Comment (michael-lee-mulesoft) [inline charts/influxdb-enterprise/values.yaml:230]: What maxseries do we have in our cloud version? I don't think we have such a limit.

**Review by rawkode — CHANGES_REQUESTED** [review]
Default value should be 1000000

This is the default when omitted

File: charts/influxdb-enterprise/values.yaml (hunk 1 above)
Comment (timhallinflux) [reply charts/influxdb-enterprise/values.yaml:230]: 1000000

**Comment by rawkode** [comment]
No update since Nov 10th, 2020. Ping me if you'd like to get this going again.

**Pull request closed without merging**